### ISSUE TITLE
Use improved learning rate schedule when training NMT jobs

### ISSUE THREAD

**Issue by ddaspit** [sillsdev/machine.py#116]
The learning rate schedule when training NMT jobs should be adjusted to use the following hyperparameters:

- learning rate schedule: cosine
- learning rate: 2.00E-04
- warmup steps: 1000
- max steps: 5000

**Comment by ddaspit** [comment]
Here is the build options for the new learning rate schedule:

``` json
{
    "train_params":
    {
        "warmup_steps": 1000,
        "learning_rate": 0.0002,
        "lr_scheduler_type": "cosine",
        "max_steps": 5000
    }
}
```
